Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        const string FirstNameMessage = "Enter your first name: ";
+        const string LastNameMessage = "Enter your last name: ";
+        const string AgeMessage = "Enter your age: ";
+        const string WeightMessage = "Enter your weight (in kg): ";
+        const string HeightMessage = "Enter your height (in meters): ";
+        
+        static string consoleInputWeight;
+        static string consoleInputHeight;
+
+        public static void Demo()
+        {            
+            string firstName = PromptString(FirstNameMessage);
+            string lastName = PromptString(LastNameMessage);
+            int age = PromptInt(AgeMessage);
+            float weight = PromptFloat(WeightMessage);
+            float height = PromptFloat(HeightMessage);
+            float bmi = CalculateBmi(weight, height);
+
+            const string Response = "{0} {1} is {2} years old, he has a BMI of {3} with a weight of {4} kg and a height of {5} meters.";
+            if (bmi > 0) Console.WriteLine(Response, firstName, lastName, age, bmi, weight, height);
```

Comment:
I like your use of response here.
However I find it irritating that you wrote a local constant with an uppercase letter. Was this suggested by Codacy?

Reply:
No. That was me. :) I will fix.